renovate.json

```diff
@@ -87,7 +87,8 @@
         "argoproj-labs/argocd-image-updater",
         "argoprojlabs/argocd-extension-installer",
         "public.ecr.aws/bitnami/redis-exporter",
-        "public.ecr.aws/docker/library/redis"
+        "public.ecr.aws/docker/library/redis",
+        "ghcr.io/dexidp/dex"
```

Comment:
The original PR doesn't seem to be run bump script. This helps enough ...? 🤔 
https://github.com/argoproj/argo-helm/pull/3172
<img width="953" alt="image" src="https://github.com/user-attachments/assets/1658a7cf-5fda-45c6-91cb-24d1ec512a41" />